src/osgUtil/SceneView.cpp

```diff
@@ -735,15 +735,13 @@ void SceneView::cull()
             _cullVisitorLeft->setClampProjectionMatrixCallback(_cullVisitor->getClampProjectionMatrixCallback());
             _cullVisitorLeft->setTraversalMask(_cullMaskLeft);
             computeLeftEyeViewport(getViewport());
-            bool computeNearFar = cullStage(computeLeftEyeProjection(getProjectionMatrix()),computeLeftEyeView(getViewMatrix()),_cullVisitorLeft.get(),_stateGraphLeft.get(),_renderStageLeft.get(),_viewportLeft.get());
-
 
             // set up the right eye.
             _cullVisitorRight->setDatabaseRequestHandler(_cullVisitor->getDatabaseRequestHandler());
             _cullVisitorRight->setClampProjectionMatrixCallback(_cullVisitor->getClampProjectionMatrixCallback());
             _cullVisitorRight->setTraversalMask(_cullMaskRight);
             computeRightEyeViewport(getViewport());
-            computeNearFar = cullStage(computeRightEyeProjection(getProjectionMatrix()),computeRightEyeView(getViewMatrix()),_cullVisitorRight.get(),_stateGraphRight.get(),_renderStageRight.get(),_viewportRight.get());
+            bool computeNearFar = cullStage(computeRightEyeProjection(getProjectionMatrix()),computeRightEyeView(getViewMatrix()),_cullVisitorRight.get(),_stateGraphRight.get(),_renderStageRight.get(),_viewportRight.get());
```

Comment:
Errh.... you've just removed the cull traversal for the left eye, this is a massive bug.  All for quietening down a benign and silly cppcheck warning.  You can NEVER EVER EVER assume that tools like cppcheck or Coverity actually know what you code is doing, you have to throughly read the code and understand what each part does and why before you can access whether it's a positive or a false positive, and then you still need to understand the code if it is a positive and you need to amend.

This bug is a good example of what it's a bad practice to bundle sweeping changes all together in a single commit and especially PR.  You have to be really thorough in reviewing code to spot potential bugs, really bug ones can lurk in innocent looking changes.

Reply:
Arg I thought I have only deleted the computeNearFar variable! this was my intention, my bad!